src/dynamics/mpas/driver/dyn_mpas_subdriver.F90

```diff
@@ -11,16 +11,25 @@ module dyn_mpas_subdriver
 
     ! Modules from external libraries.
     use mpi, only: mpi_comm_null, mpi_comm_rank, mpi_success
+    use netcdf, only: nf90_char, nf90_int, nf90_float, nf90_double
```

Comment:
Yes, I pushed https://github.com/ESCOMP/CAM-SIMA/pull/267/commits/3308a26ac86d5162b0369cecca487bef6454dfa1 to resolve this.

The reason that I used type constants from `netcdf` is due to the following.

`pio_inq_vartype` is a generic interface [[1]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/flib/pio_nf.F90#L247). Eventually, the `inq_vartype_id` function [[2]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/flib/pio_nf.F90#L1333) is called, and then the `PIOc_inq_vartype` C function [[3]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/clib/pio_nc.c#L1031) is called through ISO C binding.

`PIOc_inq_vartype` just calls the `PIOc_inq_var` function [[4]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/clib/pio_nc.c#L848), and finally, the `nc_inq_var` function [[5]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/clib/pio_nc.c#L932) from NetCDF is used to actually determine the variable type.

The PIO type constants in Fortran are defined separately [[6]](https://github.com/NCAR/ParallelIO/blob/pio2_6_2/src/flib/pio_types.F90#L133). Therefore, we can come to the conclusion that the type constants returned by `pio_inq_vartype` are entirely determined by NetCDF, and have nothing to do with the PIO type constants.

If PIO type constants were to become out-of-sync with NetCDF type constants, it would be difficult to debug.